web/pgadmin/static/js/components/EmptyPanelMessage.jsx

```diff
@@ -1,28 +1,28 @@
 import React from 'react';
+import { styled } from '@mui/material/styles';
 import { Box } from '@mui/material';
 import InfoRoundedIcon from '@mui/icons-material/InfoRounded';
-import { makeStyles } from '@mui/styles';
 import PropTypes from 'prop-types';
 
-const useStyles = makeStyles((theme)=>({
-  root: {
+const StyledBox = styled(Box)(({theme}) => ({
+  '&.EmptyPanelMessage-root': {
     color: theme.palette.text.primary,
     margin: '24px auto 12px',
     fontSize: '0.8rem',
     display: 'flex',
     alignItems: 'center',
     justifyContent: 'center',
     height: '100%',
-  },
+  }
 }));
 
 export default function EmptyPanelMessage({text, style}) {
-  const classes = useStyles();
+
   return (
-    <Box className={classes.root} style={style}>
+    <StyledBox className='EmptyPanelMessage-root' style={style}>
```

Comment:
no need to add class to root if it is not conditional

Reply:
Done.